openedx/core/djangoapps/credit/tests/test_api.py

```diff
@@ -316,10 +330,12 @@ def test_satisfy_all_requirements(self):
         ]
         api.set_credit_requirements(self.course_key, requirements)
 
+        user = UserFactory.create(username=self.USER_INFO['username'], password=self.USER_INFO['password'])
+
         # Satisfy one of the requirements, but not the other
-        with self.assertNumQueries(7):
+        with self.assertNumQueries(13):
```

Comment:
I'm surprised to see the query counts increase, based on your changes.  If this is coming from `get_account_settings` then maybe we should just pass the `User` object from the view down to this function (rather than the username).